.github/workflows/espresso-pages.yml

```diff
@@ -0,0 +1,42 @@
+name: Deploy frontend to github pages
+on:
+  push:
+    branches:
+      - "integration" # our default branch for nitro forks
+      - "release-*"
+  pull_request:
+  schedule:
+    - cron: "0 0 * * 1"
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ (github.ref == 'refs/heads/integration' && github.run_number) || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  pages:
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
+
+      - name: Build
+        run: |
+          yarn
+          yarn build
+        env:
+          # Go to https://cloud.walletconnect.com/sign-up and create a project to obtain an ID
+          NEXT_PUBLIC_WALLETCONNECT_PROJECT_ID: ${{ secrets.NEXT_PUBLIC_WALLETCONNECT_PROJECT_ID }}
+          NEXT_PUBLIC_SEPOLIA_RPC_URL: https://ethereum-sepolia-rpc.publicnode.com
+
+      - name: Deploy
+        uses: peaceiris/actions-gh-pages@v4
+        if: ${{ github.ref == 'refs/heads/integration' }}
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          publish_dir: ./packages/arb-token-bridge-ui/build
+          cname: arbitrum-bridge.cappuccino.testnet.espresso.network
```

Comment:
@Ancient123 do you think this is a reasonable domain for the arbitrum bridge frontend for our deployment?